automation/test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+ansible-lint
```

Comment:
please remove this file , we don't run any tests now.

Reply:
done